### DISCUSSION TITLE
[월드 1.1.7] Leaderboard 생성하기

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#624] [5. Q&A]
월드 1.1.7입니다 

리더보드 생성하기 예제를 보고 테스트 해보고있습니다 
https://studio.zepeto.me/guides/leaderboard
![image](https://user-images.githubusercontent.com/50188795/163552627-37866e17-3a67-4cb1-96b1-2ca48144278e.png)

저는 근데 Get Leaderboard Info를 누르면 지금 15분째 이러고있는데 어떻게 해야 하나요? 
껏다가 켜봤는데도 계속 이렇게 진행중입니다 ~

![image](https://user-images.githubusercontent.com/50188795/163552719-752cb85a-311a-42e0-85df-3f4c01d870a8.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. https://github.com/naverz/zepeto-studio-kor/discussions/424 와 동일한 현상이며
수정 버전이 올라오기 전까지는 아래 댓글에 안내드린 방법대로 진행하시면 사용 가능합니다
https://github.com/naverz/zepeto-studio-kor/discussions/424#discussioncomment-2293491

**Reply by smilejsu82** [reply]
> 안녕하세요. #424 와 동일한 현상이며 수정 버전이 올라오기 전까지는 아래 댓글에 안내드린 방법대로 진행하시면 사용 가능합니다 [#424 (comment)](https://github.com/naverz/zepeto-studio-kor/discussions/424#discussioncomment-2293491)



[월드 1.2.0] 
감사합니다 
확인 했습니다 
![image](https://user-images.githubusercontent.com/50188795/163752595-69d81f26-b860-4fa0-9c6c-08e7fede0751.png)
